pkg/cmd/gist/clone/clone.go

```diff
@@ -93,7 +94,7 @@ func cloneRun(opts *CloneOptions) error {
 }
 
 func formatRemoteURL(hostname string, gistID string, protocol string) string {
-	if ghinstance.IsEnterprise(hostname) {
+	if ghauth.IsEnterprise(hostname) || ghauth.IsTenancy(hostname) {
```

Comment:
What's the thought here? `gists` don't work with Tenants so doesn't it makes sense to mention anything relating to tenants here?

Reply:
The thought is that this maintains the exact functionality that existed before, because `ghinstance.IsEnterprise(hostname)` = `ghauth.IsEnterprise(hostname) || ghauth.IsTenancy(hostname)`

I'm fine removing it, but I thought this might save us some future headaches if gists come to Proxima. What do you think?

Reply:
I don't think we **know** what the correct behaviour would be for tenants. There's no intention behind the current behaviour, since gists never worked, it's just happenstance. If you made me wager though, I'd say that the behaviour would more likely be in line with dotcom than GHES.

What I think is happening here is that GHES uses the `gists` path prefix (in the same way it uses the `api` path prefix) to avoid having subdomain SSL certs. Tenants align with dotcom in using subdomains.